package.json

```diff
@@ -21,10 +21,12 @@
     "lint:ci": "eslint ."
   },
   "dependencies": {
-    "node-fetch": "^3.0.0-beta.9"
+    "node-fetch": "^3.0.0-beta.9",
+    "typescript": "^4.3.5"
```

Comment:
Why is this a dependency instead of a devDependency?

Reply:
thanks, was not intended